site/frontend/templates/pages/dashboard.html

```diff
@@ -1,5 +1,15 @@
 {% extends "layout.html" %}
 {% block content %}
+<details style="margin-top: 10px;">
+<summary>What data is in the dashboard?</summary>
+
+The dashboard shows performance results for all stable Rust releases going back to
+<code>1.28.0</code>, along with the latest <code>beta</code> release. The displayed
+duration is an average amongst all
```

Comment:
I'd change "average" to "arithmetic mean" to avoid the inevitable follow-up questions :)